chat_client/requirements.txt

```diff
@@ -0,0 +1,3 @@
+protobuf
```

Comment:
It's always better to specify version of libraries

Reply:
Done